integration-tests/showViewSolutionPage.test.js

```diff
@@ -0,0 +1,150 @@
+  pageSetup(t);
+  const foundationTag = Selector('div[data-test-id="solution-foundation-tag"]');
+  await t
+    .expect(foundationTag.exists).ok()
+    .expect(foundationTag.innerText).eql('Foundation Solution');
+});
+
+test('should display the organisation name', async (t) => {
+  pageSetup(t);
+  const orgName = Selector('h6[data-test-id="view-solution-page-organisation-name"]');
+  await t
+    .expect(orgName.exists).ok()
+    .expect(orgName.innerText).eql('Really Kool Corporation');
+});
+
+test('should display the solution name', async (t) => {
+  pageSetup(t);
+  const solutionName = Selector('h1[data-test-id="view-solution-page-solution-name"]');
+  await t
+    .expect(solutionName.exists).ok()
+    .expect(solutionName.innerText).eql('Write on Time');
+});
+
+test('should display the solution id', async (t) => {
+  pageSetup(t);
+  const solutionId = Selector('h5[data-test-id="view-solution-page-solution-id"]');
+  await t
+    .expect(solutionId.exists).ok()
+    .expect(solutionId.innerText).eql('100000-001');
+});
+
+test('should display the last updated', async (t) => {
+  pageSetup(t);
+  const lastUpdated = Selector('div[data-test-id="view-solution-page-last-updated"]');
+  await t
+    .expect(lastUpdated.exists).ok()
+    .expect(lastUpdated.innerText).eql('a date');
+});
+
+test('should display the solution description', async (t) => {
+  pageSetup(t);
+  const solutionDescription = Selector('div[data-test-id="view-solution-description"]');
+  const solutionDescriptionSummary = Selector('div[data-test-id="view-section-question-summary"]');
+  const solutionDescriptionDescription = Selector('div[data-test-id="view-section-question-description"]');
+  const solutionDescriptionLink = Selector('div[data-test-id="view-section-question-link"]');
+  await t
+    .expect(solutionDescription.exists).ok()
+    .expect(solutionDescription.find('h3').innerText).eql('Solution description')
+    .expect(solutionDescriptionSummary.find('div').innerText).eql('Summary')
+    .expect(solutionDescriptionSummary.find('div[data-test-id="view-question-data-text-summary"]').innerText).eql('Write on Time is a Citizen-facing Appointments Management system specifically designed to reduce the number of DNAs in your practice.')
+    .expect(solutionDescriptionDescription.find('div').innerText).eql('About the solution')
+    .expect(solutionDescriptionDescription.find('div[data-test-id="view-question-data-text-description"]').innerText).eql('a description')
+    .expect(solutionDescriptionLink.innerText).eql('link.com');
+});
+
+test('should navigate to the link address when clicking on the link in the solution description', async (t) => {
+  pageSetup(t);
+  const getLocation = ClientFunction(() => document.location.href);
+  const solutionDescriptionLink = Selector('div[data-test-id="view-section-question-link"]');
+  await t
+    .click(solutionDescriptionLink)
+    .expect(getLocation())
+    .contains('link.com');
+});
```

Comment:
Does the test try to render link.com?

Reply:
what do you mean by render?

Reply:
Is there a website at link.com, do you see that page in the test browser